homeassistant/components/hue/binary_sensor.py

```diff
@@ -26,3 +26,15 @@ class HuePresence(GenericZLLSensor, BinarySensorDevice):
     def is_on(self):
         """Return true if the binary sensor is on."""
         return self.sensor.presence
+
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        attributes = super().device_state_attributes
+        if 'sensitivity' in self.sensor.config:
+            attributes.update(
```

Comment:
Don't use `update` to update a single key, use `attributes['sensitivity'] = ` instead

Reply:
I made the change as requested